Libs/Mesh/CMakeLists.txt

```diff
@@ -41,10 +41,6 @@ install(TARGETS Mesh
  LIBRARY DESTINATION lib
  PUBLIC_HEADER DESTINATION include
  )
- 
-ADD_EXECUTABLE(GroomFemur GroomFemur.cpp)
-TARGET_LINK_LIBRARIES(GroomFemur ${ITK_LIBRARIES} Mesh tinyxml)
-INSTALL(TARGETS GroomFemur RUNTIME DESTINATION bin)
```

Comment:
We are removing this program, GroomFemur?  Can we delete GroomFemur.cpp as well then?

Reply:
May be not at this point. I think we should wait till @cchriste and @archanasri consolidate relevant mesh command so that we can re-construct this grooming workflow for meshes.

Reply:
I was going to remove it, but Oleks said to keep it cause @cchriste and @archanasri might use it somehow at some point.